provider-directory-tests/src/test/java/gov/va/api/health/providerdirectory/tests/DataQueryIT.java

```diff
@@ -28,6 +30,19 @@ static void setup() {
     assumeEnvironmentNotIn(Environment.LOCAL);
   }
 
+  @Test
+  void doNotReplaceNamingSystemUrl() {
+    final String FHIR_STRING =
```

Comment:
This is named like a constant, but it's still just a local variable for this one test. How about `var clinicIdentifierUrl`?